adapters/repos/db/vector/hnsw/search.go

```diff
@@ -693,11 +691,6 @@ func (h *hnsw) knnSearchByVector(ctx context.Context, searchVec []float32, k int
 	}
 
 	useAcorn, _ := h.acornParams(allowList)
-
-	if allowList != nil && useAcorn {
-		allowList = NewFastSet(allowList)
```

Comment:
We need to test on large environment if we should remove the FastSet. Would be nice to use an environment var for determining this for testing.